server/routes/template.js

```diff
@@ -1,10 +1,28 @@
+import config from 'config'
+
 export default (content, helmet) => `
 <!doctype html>
 <html lang="en">
   <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js"></script>
+    <script>
+      WebFont.load({
+        google: {
+          families: ['Open+Sans:400italic,700,400:latin,latin-ext']
+        }
+      });
+    </script>
     ${helmet.meta.toString()}
     ${helmet.link.toString()}
-    <link rel="stylesheet" type="text/css" href="/bundle/main.css"/>
+    <script>
```

Comment:
I'm with you on the outboundlinks but I don't understand why for file downloads?